apps/dh/api-dh/project.json

```diff
@@ -17,7 +17,7 @@
         ],
         "parallel": false
       },
-      "outputs": ["apps/dh/api-dh/source/DataHub.WebApi/bin/Release/net6.0"]
+      "outputs": ["{projectRoot}/source/DataHub.WebApi/bin/Release/net6.0"]
```

Comment:
Question: are we sure this is right?

Reply:
I think it is. You can see the migration [here](https://github.com/nrwl/nx/pull/12470/files#diff-c8817fc8a17002f12e5c8565ad91616142cd2ccdd965c47b18bb0bcf7d9f442e). It looks like if the path has something to do with the current project, then it's prefixed with `{projectRoot}`. On the other hand, if the path has something to do with another project in the workspace, it's prefixed with `{workspaceRoot}`.